### ISSUE TITLE
BUILD Reserve Bounties Improvements

### ISSUE THREAD

**Issue by thelostone-mc** [gitcoinco/web#3544]
##### Description

As of now, a funder can come reserve a bounty for a user `X` while creating a bounty `/new`.
The created bounty is reserved for `X` for upto 72 hours during which only he can start work on it.
After that -> the issue is no longer reserved and is open to everyone to work on.

PR where it was built out: https://github.com/gitcoinco/web/pull/2950

###### Issues 

- Right now, a reserved issue pops up into the `/explorer` for everyone even though they cannot start work on it.
- We have a simple check to determine if a reserved issue is [open](https://github.com/gitcoinco/web/blob/72638e148b87854c648e120e3e6e05c42fcd6dbc/app/assets/v2/js/pages/bounty_details.js#L431-L445) to the public, this means even after 72 hours expire -> we maintain who the issue was reserved for.
If the grant owner edits the issue details -> the checks kick in and the issue is reserved again (which should not happen )
- There are instances where the funder may want the reserved issue to go beyond 72 hours (3 days)  and that doesn't happen at the moment.

##### PR should Fix

- Introduce input on the same row as reserved for in `/new` which asks 
 `Release to the public after` X with drop down with options  `after 3 days, after a week, keep reserved` which map to corresponding hours 
- Introduce a new state -> reserved. 
- Any bounty created which is  reserved for a user -> would be in this newly created state
- If the reserved bounty is started by user -> issue would transition to `Work Started`
- If the bounty is no longer reserved (aka reserved time elapses) remove the reserved for User from db & transition issue to `Open State`
- Add new state to the filter on the `explorer` to enable logged in user to see bounties reserved for them
- ensure reserved bounty appears on `explorer` only to the users who have an active reservation
- provide and button on the issue detail to force release the issue if needed. (Aka update dp to remove the reservedFor Data ) 

Discussions : https://github.com/gitcoinco/web/issues/3502

##### Note

This task would undergo PR reviews and feedback would have to be worked upon

**Comment by PixelantDesign** [comment]
If the funder has reserved it for a contributor, we would undo the reserved feature after 72 hours?

**Comment by thelostone-mc** [comment]
@PixelantDesign Yes, the code snippet which does that is mentioned in the description ^_^

**Comment by PixelantDesign** [comment]
My recommendation is to add a checkbox and let the user opt into the 72 hour realse.... instead of make assumptions that users would want it undone when they’ve clearly set it to reserve.

**Comment by thelostone-mc** [comment]
@PixelantDesign Updated the ticket. Freezing the scope for this ticket. 🙌

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__This issue now has a funding of 0.45 ETH (53.86 USD @ $119.69/ETH)  attached to it.__

 * If you would like to work on this issue you can 'start work' [on the Gitcoin Issue Details page](https://gitcoin.co/issue/gitcoinco/web/3544/2128).
* Want to chip in? Add your own contribution [here](https://gitcoin.co/issue/gitcoinco/web/3544/2128).
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
* $40,214.39 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__Workers have applied to start work__.


These users each claimed they can complete the work by 4 weeks, 1 day from now.
Please review their action plans below:


**1) [usmanmuhd](https://gitcoin.co/profile/usmanmuhd) has applied to start work _(Funders only: [approve worker](https://gitcoin.co/issue/gitcoinco/web/3544/2128?mutate_worker_action=approve&worker=usmanmuhd) | [reject worker](https://gitcoin.co/issue/gitcoinco/web/3544/2128?mutate_worker_action=reject&worker=usmanmuhd))_.**

WIll be doing this as discussed with @thelostone-mc before.

Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/gitcoinco/web/3544/2128).

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. **Started** 3. Submitted 4. Done 

<hr>

__Work has been started__.


These users each claimed they can complete the work by 4 weeks, 1 day from now.
Please review their action plans below:


**1) [usmanmuhd](https://gitcoin.co/profile/usmanmuhd) has been approved to start work.**

WIll be doing this as discussed with @thelostone-mc before.

Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/gitcoinco/web/3544/2128).

**Comment by gitcoinbot** [comment]
@usmanmuhd Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=100)

**Comment by PixelantDesign** [comment]
Yes and contributors who cannot apply to work on this should not get notifications of this new issue.

**Comment by gitcoinbot** [comment]
@usmanmuhd Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=100)

**Comment by gitcoinbot** [comment]
@usmanmuhd Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=100)

**Comment by gitcoinbot** [comment]
@usmanmuhd Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=100)

**Comment by gitcoinbot** [comment]
@usmanmuhd Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=100)

**Comment by gitcoinbot** [comment]
@usmanmuhd Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=100)

**Comment by thelostone-mc** [comment]
@usmanmuhd could we have the PR up for this / need help from my side ?

**Comment by gitcoinbot** [comment]
@usmanmuhd Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=100)

**Comment by thelostone-mc** [comment]
@usmanmuhd could we get a WIP PR opened up today ? I would like this in by this week

**Comment by usmanmuhd** [comment]
Yeah sure. Today evening it'll be done.

**Comment by thelostone-mc** [comment]
@usmanmuhd still waiting on the PR 😓

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__Workers have applied to start work__.


These users each claimed they can complete the work by 2 weeks, 3 days from now.
Please review their action plans below:


**1) [satyamakgec](https://gitcoin.co/profile/satyamakgec) has applied to start work _(Funders only: [approve worker](https://gitcoin.co/issue/gitcoinco/web/3544/2128?mutate_worker_action=approve&worker=satyamakgec) | [reject worker](https://gitcoin.co/issue/gitcoinco/web/3544/2128?mutate_worker_action=reject&worker=satyamakgec))_.**

1. Code setup
2. Understand the Code
3. Finding the pinpoint file 
4. Implement the changes
5. Fix the test cases

Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/gitcoinco/web/3544/2128).

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. **Started** 3. Submitted 4. Done 

<hr>

__Work has been started__.


These users each claimed they can complete the work by 2 weeks, 3 days from now.
Please review their action plans below:


**1) [satyamakgec](https://gitcoin.co/profile/satyamakgec) has been approved to start work.**

1. Code setup
2. Understand the Code
3. Finding the pinpoint file 
4. Implement the changes
5. Fix the test cases

Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/gitcoinco/web/3544/2128).

**Comment by gitcoinbot** [comment]
@satyamakgec Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=100)

**Comment by gitcoinbot** [comment]
@satyamakgec Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=100)

**Comment by satyamakgec** [comment]
@gitcoinbot yes I am working on the issue.

**Comment by gitcoinbot** [comment]
@satyamakgec Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=100)

**Comment by gitcoinbot** [comment]
@satyamakgec Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=100)

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. **Started** 3. Submitted 4. Done 

<hr>

@satyamakgec due to inactivity, we have escalated [this issue](https://gitcoin.co/issue/gitcoinco/web/3544/2128) to Gitcoin's moderation team. Let us know if you believe this has been done in error!

* [x] reminder (3 days)
* [x] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=100)

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. **Started** 3. Submitted 4. Done 

<hr>

@satyamakgec due to inactivity, we have escalated [this issue](https://gitcoin.co/issue/gitcoinco/web/3544/2128) to Gitcoin's moderation team. Let us know if you believe this has been done in error!

* [x] reminder (3 days)
* [x] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=100)

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__The funding of this issue was increased to 0.7 ETH (178.87 USD @ $255.53/ETH) .__

 
* If you want to claim the bounty you can do so [here](https://gitcoin.co/issue/gitcoinco/web/3544/2128)
* Want to chip in? Add your own contribution [here](https://gitcoin.co/issue/gitcoinco/web/3544/2128).
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
* $84,643.67 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__Workers have applied to start work__.


These users each claimed they can complete the work by 1 month, 4 weeks ago.
Please review their action plans below:


**1) [rafalkowalski](https://gitcoin.co/profile/rafalkowalski) has applied to start work _(Funders only: [approve worker](https://gitcoin.co/issue/gitcoinco/web/3544/2128?mutate_worker_action=approve&worker=rafalkowalski) | [reject worker](https://gitcoin.co/issue/gitcoinco/web/3544/2128?mutate_worker_action=reject&worker=rafalkowalski))_.**

I would like to work on bounty reservetion improvements

Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/gitcoinco/web/3544/2128).

**Comment by gitcoinbot** [comment]
@rafalkowalski Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=100)

**Comment by gitcoinbot** [comment]
@rafalkowalski Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=100)

**Comment by gitcoinbot** [comment]
@rafalkowalski Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=100)

**Comment by gitcoinbot** [comment]
@rafalkowalski Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=100)

**Comment by gitcoinbot** [comment]
@rafalkowalski Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=100)

**Comment by gitcoinbot** [comment]
@rafalkowalski Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=100)

**Comment by gitcoinbot** [comment]
@rafalkowalski Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=100)

**Comment by gitcoinbot** [comment]
@rafalkowalski Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=100)

**Comment by gitcoinbot** [comment]
@rafalkowalski Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=100)

**Comment by PixelantDesign** [comment]
@rafalkowalski could you provide an update on this?

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__Workers have applied to start work__.


These users each claimed they can complete the work by 2 weeks from now.
Please review their action plans below:


**1) [vince0656](https://gitcoin.co/profile/vince0656) has applied to start work _(Funders only: [approve worker](https://gitcoin.co/issue/gitcoinco/web/3544/2128?mutate_worker_action=approve&worker=vince0656) | [reject worker](https://gitcoin.co/issue/gitcoinco/web/3544/2128?mutate_worker_action=reject&worker=vince0656))_.**

Hi there,

Having worked on Gitcoin before and added new bounty functionality, I feel like I could give this a good shot. I will aim to fix the issues listed in the description which includes:
-Release to public feature
-New state of reserved
-New bounties created with reservations are marked as reserved
-Transition to work started for reserved user that starts work
-Transition to open state if reserved user doesn't start working on issue after X time
-New filter to see bounties reserved for users

Any questions, let me know - thanks!

Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/gitcoinco/web/3544/2128).

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. **Started** 3. Submitted 4. Done 

<hr>

__The funding of this issue was increased to 0.98 ETH (180.54 USD @ $184.22/ETH) .__

 
* If you want to claim the bounty you can do so [here](https://gitcoin.co/issue/gitcoinco/web/3544/2128)
* Want to chip in? Add your own contribution [here](https://gitcoin.co/issue/gitcoinco/web/3544/2128).
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
* $138,841.45 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by gitcoinbot** [comment]
@vince0656 Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/3544/2128?snooze=100)

**Comment by vince0656** [comment]
Hi there,

I'm still working on this.

Thanks very much,
Vincent

**Comment by vince0656** [comment]
Hi there,

I'm still working on this.

Thanks very much,
Vincent

**Comment by vince0656** [comment]
Hi there,

I'm still working on this. Coming to the end now 

Thanks very much,
Vincent

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. Started 3. **Submitted** 4. Done 

<hr>

__Work for 0.98 ETH (166.4 USD @ $169.79/ETH) has been submitted by__:

 1. [@vince0656](https://gitcoin.co/profile/vince0656)



@PixelantDesign please take a look at the submitted work:
* [PR](https://github.com/gitcoinco/web/pull/5066) by @vince0656

<hr>

* Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/gitcoinco/web/3544/2128)
* Want to chip in? Add your own contribution [here](https://gitcoin.co/issue/gitcoinco/web/3544/2128).
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
* $133,098.99 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by thelostone-mc** [comment]
@PixelantDesign we are good for payout !

**Comment by vince0656** [comment]
@PixelantDesign @thelostone-mc @octavioamu Sorry to bug but any idea as to when this will be paid out? PR merged into master is: #5066 

Thanks in advance

**Comment by owocki** [comment]
looks like @PixelantDesign owns this; and she just went out on maternity leave!  

i can pay you out of my funds if you want ... then when @PixelantDesign gets back we can cancel the bounty

**Comment by vince0656** [comment]
@owocki yes please. Could you pay me and then cancel the bounty when @PixelantDesign is back. Thanks

**Comment by vince0656** [comment]
@owocki any update on the payout? Thanks in advance

**Comment by owocki** [comment]
sorry; devcon broke me inbox! paying out now

**Comment by gitcoinbot** [comment]
⚡️ A tip worth 0.98000  ETH (182.57 USD @ $186.3/ETH) has been granted to @vince0656 for this issue from @owocki. ⚡️ 

Nice work @vince0656! Your tip has automatically been deposited in the ETH address we have on file.

 * $150189.58 in Funded OSS Work Available at: https://gitcoin.co/explorer
 * Incentivize contributions to your repo: <a href='https://gitcoin.co/tip'>Send a Tip</a> or <a href='https://gitcoin.co/funding/new'>Fund a PR</a>
 * No Email? Get help on the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>

**Comment by owocki** [comment]
@PixelantDesign when ur back from leave; pls cancel the bounty (do not payout; i already paid!)
